includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -1106,4 +1107,29 @@ public function update_content_via_rest( $new_post_id ) {
 			);
 		}
 	}
+
+	/**
+	 * Exclude additional meta data for network distributions
+	 *
+	 * @hook dt_exclude_additional_meta_data
+	 *
+	 * In network connections the featured image is set prior to the meta data.
+	 * Excluding the `_thumbnail_id` meta from distribution prevents the meta
+	 * data from referring to the attachment ID of the original site.
+	 *
+	 * You can use this filter to exclude additional meta data from the distribution.
+	 *
+	 * @since 2.0.2
+	 *
+	 * @param string[] $post_meta Array of meta to include in the distribution.
+	 * @return string[] Array of meta to include in the distribution after filtering out excluded meta.
+	 */
+	public static function exclude_additional_meta_data( $post_meta ) {
+		$keys_to_exclude = array();
+		$keys_to_exclude[] = '_thumbnail_id';
+		$keys_to_exclude = apply_filters( 'dt_exclude_additional_meta_data', $keys_to_exclude );
+		$modified_meta = array_diff_key( $post_meta, array_flip( $keys_to_exclude ) );
+
+		return $modified_meta;
```

Comment:
This is much nicer than my suggestion, we can probably simplify it to the following and decide whether to add the hook later -- that will allow us to add `excluded_meta_internal` and `excluded_meta_external` at the same time.

```suggestion
		unset( $post_meta['_thumbnail_id'] );
		return $post_meta;
```

`unset()` doesn't throw if the array key doesn't exist so doesn't need an `isset()` or equivalent.